runtime-interfaces/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "aleph-runtime-interfaces"
+version = "0.1.0"
+authors = ["Cardinal", "Aleph Zero Foundation"]
+edition = "2021"
+homepage = "https://alephzero.org"
+repository = "https://github.com/Cardinal-Cryptography/aleph-node"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0", default-features = false, features = ["derive"] }
+log = { version = "0.4", default-features = false }
+
+halo2_proofs = { git = "https://github.com/Cardinal-Cryptography/pse-halo2", branch = "substrate-compatible", default-features = false, features = ["mock-kzg-params"], optional = true }
```

Comment:
As discussed I believe it may be beneficial to divide this dependency into two: curves and abstractions+verifier. But it may be added in separate pr